ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,15 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.3.0-beta4 24-Oct-2022
+Changes from 4.3.0-beta3
+1) Scripting supports implementing AUX functions
+2) Bug fixes
```

Comment:
There is a fairly significant fix for supporting the harmonic notch on non-contiguous motor sets which prevents a crash. I cannot find the PR however -  the release note should be in Plane as well since it is particularly significant for plane

Reply:
The PR is this one https://github.com/ArduPilot/ardupilot/pull/21473

Reply:
ok, thanks added.